src/features/public/Donations/components/ContactDetails.tsx

```diff
@@ -214,7 +215,10 @@ function ContactDetails({
 
         <div className={styles.finalTreeCount}>
           <div className={styles.totalCost}>
-            {currency} {Sugar.Number.format(Number(treeCount * treeCost), 2)}
+          {t('donate:fortreeAmountTrees', {
+            amount: getFormatedCurrency(i18n.language, currency, treeCount * treeCost),
```

Comment:
`{ getFormatedCurrency(i18n.language, currency, treeCount * treeCost) }` should be enough here?

Reply:
ok. i have made the changes